### PR TITLE
Fiche article: add field species and page Species Information

### PR DESCRIPTION
**Pull request by robinkeunen**
ref: [2 - Fiche article : adaptation excepté infos variété](https://gestion.coopiteasy.be/web#id=1307&view_type=form&model=project.task&action=479)
ref: [3 - A1 - Fiche article : web info -> infos variété](https://gestion.coopiteasy.be/web#id=1308&view_type=form&model=project.task&action=479)

### CODE REVIEW

**Review by remytms — CHANGES_REQUESTED** [review]
Looks good to me. Just correct the line ending and pass Travis checks. :)

File: product_seeds/data/data.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <record id="january" model="seedling.month">
```
Comment (houssine78) [inline product_seeds/data/data.xml:4]: default value of noupdate is False so except if it's True you can't avoid putting it

**Review by houssine78 — COMMENTED** [review]
LGTM

File: product_seeds/data/data.xml (hunk 1 above)
Comment (robinkeunen) [reply product_seeds/data/data.xml:4]: True

**Pull request merged**